### PR TITLE
Feature/#2795 translation ubs admin order

### PR DESCRIPTION
**Pull request by YuriShevchuk**
Add changes UBS Admin order page

- add translation to ubs-admin-order
- some refactor forms

### CODE REVIEW

File: src/app/ubs-admin/components/ubs-admin-address-details/ubs-admin-address-details.component.html [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
   <form [formGroup]="addressDetailsForm">
     <div class="form-row">
       <div class="form-group col-md-6">
-        <label for="street">Вулиця</label>
+        <label for="street">{{ "address-details.street" | translate }}</label>
         <input type="text" formControlName="street" class="form-control" id="street" />
       </div>
       <div class="form-group col-md-5 offset-md-1">
-        <label for="addess-comment">Коментар до адреси від клієнта</label>
+        <label for="addess-comment">{{ "address-details.comment-label" | translate }}</label>
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-address-details/ubs-admin-address-details.component.html:12]: Rename  'addess-comment' to 'address-comment'

File: src/app/ubs-admin/components/ubs-admin-export-details/ubs-admin-export-details.component.html [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
       <input type="date" class="form-control" />
     </div>
     <div class="form-group col-md-2 mx-4">
-      <label>Час вивезення</label>
+      <label>{{ "export-details.export-time" | translate }}</label>
       <input type="time" class="form-control" />
     </div>
     <div class="form-group col-md-4">
-      <label>Станція приймання</label>
+      <label>{{ "export-details.station" | translate }}</label>
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-export-details/ubs-admin-export-details.component.html:13]: add 'for' attribute to label and 'id' to inputs

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html [hunk 3]
```diff
@@ -1,53 +1,53 @@
...
     </div>
     <div class="form__wrapper row d-flex">
       <div class="form__block col-md-4 col-sm-6">
         <div class="form-group">
-          <p class="form__subtitle">Клієнт</p>
-          <label class="form__label" for="customer-name">Ім'я</label>
-          <input class="form-control" type="text" id="customer-name" placeholder="Placeholder" formControlName="customerName" />
+          <p class="form__subtitle">{{ "customer-info.customer.title" | translate }}</p>
+          <label class="form__label" for="customer-name">{{ "customer-info.customer.name-label" | translate }}</label>
+          <input class="form-control" type="text" id="customer-name" placeholder="{{ 'customer-info.customer.name-placeholder' | translate }}" formControlName="customerName" />
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html:11]: Break attributes to new lines

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html [hunk 4]
```diff
@@ -1,53 +1,53 @@
...
         </div>
         <div class="form-group">
-          <label class="form__label" for="customer-tel">Телефон</label>
-          <input class="form-control" type="tel" id="customer-tel" placeholder="Placeholder" formControlName="customerPhoneNumber" />
+          <label class="form__label" for="customer-tel">{{ "customer-info.customer.tel-label" | translate }}</label>
+          <input class="form-control" type="tel" id="customer-tel" placeholder="{{ 'customer-info.customer.tel-placeholder' | translate }}" formControlName="customerPhoneNumber" />
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html:15]: Break attributes to new lines

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html [hunk 5]
```diff
@@ -1,53 +1,53 @@
...
         </div>
         <div class="form-group">
-          <label class="form__label" for="customer-email">Email</label>
-          <input class="form-control" type="email" id="customer-email" placeholder="Placeholder" formControlName="customerEmail" />
+          <label class="form__label" for="customer-email">{{ "customer-info.customer.email-label" | translate }}</label>
+          <input class="form-control" type="email" id="customer-email" placeholder="{{ 'customer-info.customer.email-placeholder' | translate }}" formControlName="customerEmail" />
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html:19]: Break attributes to new lines

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html [hunk 6]
```diff
@@ -1,53 +1,53 @@
...
-          <label class="form__label" for="recipient-tel">Телефон</label>
-          <input class="form-control" type="tel" id="recipient-tel" placeholder="Placeholder" formControlName="recipientPhoneNumber" />
+          <label class="form__label" for="recipient-tel">{{ "customer-info.recipient.tel-label" | translate }}</label>
+          <input class="form-control" type="tel" id="recipient-tel" placeholder="{{ 'customer-info.recipient.tel-placeholder' | translate }}" formControlName="recipientPhoneNumber" />
         </div>
         <div class="form-group">
-          <label class="form__label" for="recipient-email">Email</label>
-          <input class="form-control" type="email" id="recipient-email" placeholder="Placeholder" formControlName="recipientEmail" />
+          <label class="form__label" for="recipient-email">{{ "customer-info.recipient.email-label" | translate }}</label>
+          <input class="form-control" type="email" id="recipient-email" placeholder="{{ 'customer-info.recipient.email-placeholder' | translate }}" formControlName="recipientEmail" />
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html:34]: Same here

File: src/app/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.html [hunk 7]
```diff
@@ -78,45 +78,45 @@ <h3 class="form__headline pl-3">Деталі замовлення</h3>
...
     <div class="bottom d-flex align-items-baseline py-3">
       <div class="form-floating col-sm-6">
-        <label for="textarea">Коментар до замовлення від клієнта</label>
+        <label for="textarea">{{ "order-details.client-comment.label" | translate }}</label>
         <textarea
           class="form-control comment"
           formControlName="customerComment"
-          placeholder="Будь-які деталі, які Ви хотіли б повідомити додатково щодо цього замовлення"
+          placeholder="{{ 'order-details.client-comment.placeholder' | translate }}"
           id="textarea"
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.html:114]: Let's rename 'id' and 'for'  to customer-comment

File: src/app/ubs-admin/components/ubs-admin-address-details/ubs-admin-address-details.component.html (hunk 1 above)
Comment (YuriShevchuk) [reply src/app/ubs-admin/components/ubs-admin-address-details/ubs-admin-address-details.component.html]: Done

File: src/app/ubs-admin/components/ubs-admin-export-details/ubs-admin-export-details.component.html (hunk 2 above)
Comment (YuriShevchuk) [reply src/app/ubs-admin/components/ubs-admin-export-details/ubs-admin-export-details.component.html]: Added

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html (hunk 3 above)
Comment (YuriShevchuk) [reply src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html]: Done

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html (hunk 4 above)
Comment (YuriShevchuk) [reply src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html]: Done

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html (hunk 5 above)
Comment (YuriShevchuk) [reply src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html]: Done

File: src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html (hunk 6 above)
Comment (YuriShevchuk) [reply src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html]: Done

File: src/app/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.html (hunk 7 above)
Comment (YuriShevchuk) [reply src/app/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.html]: Done

**Review by rostikdoroshenko — APPROVED** [review]

**Review by Edw1x — APPROVED** [review]

**Review by tkhristina — APPROVED** [review]

**Review by YulianDev — APPROVED** [review]

**Review by JimHolman1199 — APPROVED** [review]

**Review by andreikazmirovich — APPROVED** [review]

**Review by dyadyushko — APPROVED** [review]

**Pull request merged**